single-user/Dockerfile

```diff
@@ -38,24 +42,23 @@ RUN mamba create --name octave -y --quiet octave octave_kernel \
     && OCTAVE_PATH=$(conda run -n octave /bin/bash -c "echo \$PATH") \
```

Comment:
Frantisek reported an issue with Octave which is solved if installed before `conda config --add channels bioconda .`  i didn't try but we can make this change i guess